### PR TITLE
Display settings Automation

### PR DESCRIPTION
**Pull request by akarol**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 00d2326bbf4ea088c17136e73b64be4be6b4b676:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4311cffd75e0958ebb075099018ec2b63b58d128:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5e67feb5d3fc357eb6a112d36e6d63dc64e0e06d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_display_settings.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        test_flag: visuals
+    """
+    sel.force_navigate("my_settings_visual")
+    cg = ColorGroup('Header Accent Color')
+    default_color = cg.active
+    cg.choose(name)
+    sel.click(form_buttons.save)
+    assert cg.status(name), "Header Accent Color setting failed"
+    cg.choose(default_color)
+    sel.click(form_buttons.save)
```
Comment (seandst) [inline cfme/tests/configure/test_display_settings.py]: We should assert that the header color actually changed, and probably add in a fixalizer at the module scope to set it back to the default if it isn't already

**Comment by dajoRH** [comment]
Lint report for commit e674fccf24ee236b4a5e6160602b56fade3ef7f4:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ef92c89da81d522c32ddc40cf667302ffe73c2c3:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0e9dd79532d566d4f3aaa6c5cadaa76f3d268abf:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**